packages/ui/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "build": "tsup",
-    "dev": "NODE_OPTIONS='--max-old-space-size=4096' tsup  --watch",
+    "dev": "tsup --watch",
```

Comment:
I believe, Jason configured the memory size since the build is failing because of memory issue. please check if the dev is working if we remove this.